package.json

```diff
@@ -66,4 +65,4 @@
     ],
     "configPath": "tests/dummy/config"
   }
-}
\ No newline at end of file
+}
```

Comment:
You didn't update `ember-cli-babel` or `ember-cli-htmlbars in `dependencies` section